app/views/pages/about.html.erb

```diff
@@ -36,30 +36,31 @@
+      <li class="about_li"><%= link_to('Documenting the Now', 'https://www.docnow.io', target: '_blank') %></li>
+      <li class="about_li"><%= link_to('Webrecorder.io', 'https://webrecorder.io', target: '_blank') %></li>
+      <li class="about_li"><%= link_to('Internet Archive', 'https://archive.org', target: '_blank') %></li>
     </ul>
 
   <h3 class=about_h3>How much does AUK cost to use?</h3>
-    <p class="about_p">The Archives Unleashed Cloud is currently <strong>free</strong>. Yep, you heard right! The generous support from the <a href="https://mellon.org" target="_blank"> Andrew W. Mellon Foundation</a> foundation and contributions from our team, developers, and community have made this possible. We will keep our users and community updated as we move towards a sustainable future.</p>
+    <p class="about_p">The Archives Unleashed Cloud is currently <strong>free</strong>. Yep, you heard right! The generous support from the <%= link_to('Andrew W. Mellon Foundation', 'https://mellon.org', target: '_blank') %> and contributions from our team, developers, and community have made this possible. We will keep our users and community updated as we move towards a sustainable future.</p>
 
   <h3 class=about_h3>Who funds the Archives Unleashed Cloud?</h3>
-    <p class="about_p">This work is primarily supported by the <a href="https://mellon.org/" target="_blank"> Andrew W. Mellon Foundation</a>, the <a href="http://uwaterloo.ca" target="_blank">University of Waterloo</a>, and <a href="https://www.library.yorku.ca/web/" target="_blank">York University Libraries</a>.</p>
-    <p class="about_p">Other financial and in-kind support comes from the <a href="http://www.sshrc-crsh.gc.ca" target="_blank">Social Sciences and Humanities Research Council</a>, <a href="https://www.computecanada.ca" target="_blank">Compute Canada</a>, the <a href="https://www.ontario.ca/page/ministry-research-innovation-and-science" target="_blank">Ontario Ministry of Research, Innovation, and Science</a>, and <a href="http://www.startsmartlabs.com" target="_blank">Start Smart Labs</a>.</p>
+    <p class="about_p">This work is primarily supported by the <%= link_to('Andrew W. Mellon Foundation', 'https://mellon.org/', target: '_blank') %>, the <%= link_to('University of Waterloo', 'http://uwaterloo.ca', target: '_blank') %>, and  <%= link_to('York University Libraries', 'https://www.library.yorku.ca/web/', target: '_blank') %>.</p>
+    <p class="about_p">Other financial and in-kind support comes from the <%= link_to('Social Sciences and Humanities Research Council', 'http://www.sshrc-crsh.gc.ca', target: '_blank') %>, <%= link_to('Compute Canada', 'https://www.computecanada.ca', target: '_blank') %>,the <%= link_to('Ontario Ministry of Research, Innovation, and Science', 'https://www.ontario.ca/page/ministry-research-innovation-and-science', target: '_blank') %>, and <%= link_to('Start Smart Labs', 'http://www.startsmartlabs.com', target: '_blank') %>
```

Comment:
need a space after the comma after Compute Canada

i.e.
`,the`

should be

`, the`